recipe/meta.yaml

```diff
@@ -70,14 +70,16 @@ source:
 
 
 build:
-  number: 3
+  number: 4
   # Windows has issues updating python if conda is using files itself.
   # Copy rather than link.
   no_link:
     - DLLs/_ctypes.pyd  # [win]
   script_env:
     - python_branding
   skip: true  # [win and vc != '14.1']
+  skip_compile_pyc:
+    - lib/python3.*/lib2to3/tests/data/*.py  # [py>=30]
```

Comment:
There is not need for the selector here and I'm not entirely sure how it resolves.